app/models/manageiq/providers/redhat/infra_manager/refresh/parse/parser.rb

```diff
@@ -6,7 +6,7 @@ class ManageIQ::Providers::Redhat::InfraManager::Refresh::Parse::Parser
   def self.ems_inv_to_hashes(inv)
     uids = {}
     result = {:uid_lookup => uids}
-    result[:ems] = inv[:ems]
+    result[:ems] = ems_version_to_hashes(inv)
```

Comment:
why have we decided to use it?

Reply:
It was added by this commit: https://github.com/ManageIQ/manageiq-providers-ovirt/commit/3e2cdfc412a72c12cb88557dbb5ae585c8834229

However, it wasn't working properly for v4.